restcomm/restcomm.mscontrol.jsr309/src/main/java/org/restcomm/connect/mscontrol/jsr309/Jsr309CallController.java

```diff
@@ -776,7 +776,7 @@ private void onStop(Stop message, ActorRef self, ActorRef sender) {
                         logger.error("Could not measure recording duration: " + e.getMessage(), e);
                         duration = 0.0;
                     }
-                    if (duration.equals(0.0)) {
+                    if (!duration.equals(0.0)) {
```

Comment:
we just flipped the check here and did nothing further.
dont we want to add recording record inside this check to make sure its added only when recording duration is not 0